src/components/app-list-item/style.css

```diff
@@ -69,15 +68,58 @@
   content: url('!~@fortawesome/fontawesome-free/svgs/solid/cog.svg');
   display: block;
   height: 4rem;
-  margin-top: calc(-2rem + var(--space-3)  / 2); /* 1 */
+  margin-top: calc(-2rem + var(--space-3) / 2); /* 1 */
   opacity: 0.1;
   position: absolute;
   top: 50%;
   right: -2rem;
   width: 4rem;
 }
 
+@keyframes animate-placeholder {
+  0% {
+    transition: none;
+  }
+  50% {
+    background-color: var(--color-mood-darken);
+  }
+  100% {
+    transition: none;
+  }
+}
+
 .app-list-item--failed a,
 .app-list-item--failed a:hover {
   color: var(--color-danger-dark);
 }
+
+.app-list-item--placeholder {
+  animation: none;
+  opacity: 1;
+}
+
+.app-list-item--placeholder .app-list-item__area-icon {
+  animation-duration: 2s;
```

Comment:
`calc(var(--duration-slow) * 2)` or, even better, up in `.app-list-item` define a new variable:

`--placeholder-anim-duration: calc(var(--duration-slow) * 2)`

And then use that here and in line 114 below